app/models/navigation_link.rb

```diff
@@ -5,6 +5,7 @@ class NavigationLink < ApplicationRecord
   before_save :strip_local_hostname, if: :url?
 
   enum section: { default: 0, other: 1 }, _suffix: true
+  enum display_to: { all: 0, logged_in: 1, logged_out: 2 }, _suffix: true
```

Comment:
Is there a reason why you would prefer `_suffix: true` over `_prefix: true`? I have a suggestion above to leverage the `enum` helper methods Rails gives us, but it reads better with `_prefix: true` because it generates `.display_to_all?` vs `.to_all_display_to?`

```suggestion
  enum display_to: { all: 0, logged_in: 1, logged_out: 2 }, _prefix: true
```

Reply:
hahaha yes of course 🙈 I think I added this and then forgot all about it tbh - thanks for the suggestion!!